packages/amazon-sumerian-hosts-babylon/src/Babylon.js/awspack/LexFeature.js

```diff
@@ -0,0 +1,15 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+import {LexFeature as CoreLexFeature} from '@amazon-sumerian-hosts/core';
+
+/**
+ * @extends core/LexFeature
+ * @alias Babylon.js/LexFeature
+ */
+class LexFeature extends CoreLexFeature {
+  _setAudioContext() {
+    this._audioContext = BABYLON.Engine.audioEngine.audioContext;
```

Comment:
excuse my ignorance on babylon, should we be using a Babylon import instead of the static global?

Reply:
I will refactor this. Yes. I think we should use Babylon import.